README.md

```diff
@@ -73,7 +52,7 @@ the Hash type to support extra configuration.
 By default, the duplication engine will report code that has been duplicated in just two locations. You can be less strict by only raising a warning if code is duplicated in three or more locations only. To adjust this setting, add a `count_threshold` key to your config. For instance, to use the default `mass_threshold` for ruby, but to enforce the [Rule of Three][rule-of-three], you could use this configuration:
 
 ```yaml
-engines:
+plugins:
   duplication:
     enabled: true
     config:
```

Comment:
We we properly merge the plugin-style configuration with the similar/identical check configuration?

Reply:
For the purposes of legacy config support, yes.